app/main.py

```diff
@@ -1,5 +1,85 @@
+from typing import Union
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+        self,
+        name: str,
+        weight: int,
+        coords: list[int] = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        if coords is not None:
+            self.coords = coords
+        else:
+            self.coords = [0, 0]
```

Comment:
You can rewrite this by using the ternary operator
```suggestion
        self.coords = coords if coords else [0, 0]
```